cockroachcloud/create-a-serverless-cluster.md

```diff
@@ -22,7 +23,7 @@ If you haven't already, <a href="https://cockroachlabs.cloud/signup?referralId=d
 1. If there are multiple [organizations](../{{site.versions["stable"]}}/architecture/glossary.html#organization) in your account, select the correct organization in the top right corner.
 1. On the **Overview** page, click **Create Cluster**.
 
-## Step 2. Select a cloud provider & region
+## Step 2. Select the cloud provider
 
 1. _(Optional)_ Select a cloud provider (GCP or AWS) in the **Cloud provider** section.
```

Comment:
Worth noting that serverless isn't yet supported on Azure?

Reply:
Hm, maybe when we get out of the LA phase